Services/AssetsService/Package.swift

```diff
@@ -25,13 +33,37 @@ let package = Package(
                 "Primitives",
                 "Store",
                 "GemAPI",
-                "ChainService"
+                "ChainService",
+                "NodeService",
+                "Preferences",
+                "GemstonePrimitives"
             ],
             path: "Sources"
         ),
+        .target(
+            name: "AssetsServiceTestKit",
+            dependencies: [
+                .product(name: "StoreTestKit", package: "Store"),
+                .product(name: "GemAPITestKit", package: "GemAPI"),
+                .product(name: "ChainServiceTestKit", package: "ChainService"),
+                "AssetsService",
+                "Primitives",
+                "Preferences",
+                "GemstonePrimitives"
+            ],
+            path: "TestKit"
+        ),
         .testTarget(
             name: "AssetsServiceTests",
-            dependencies: ["AssetsService"]
+            dependencies: [
+                .product(name: "StoreTestKit", package: "Store"),
+                .product(name: "GemAPITestKit", package: "GemAPI"),
+                .product(name: "PreferencesTestKit", package: "Preferences"),
+                .product(name: "ChainServiceTestKit", package: "ChainService"),
+                "AssetsService",
+                "AssetsServiceTestKit",
+                "Primitives"
```

Comment:
```suggestion
                "Primitives"
```

move it up, first in order